package/libnetwork/src/libnetwork.c

```diff
@@ -520,6 +537,10 @@ static void network_request(network_context_t* ctx, CURL *curl)
       if (response != 0) {
         sbp_log(LOG_INFO, "Network Request Response Code - (%d)", response);
         piksi_log(LOG_INFO, "curl request code %d", response);
+        ctx->response_code = response;
+        if (network_response_code_check(ctx)) {
+          return;
```

Comment:
Why exit here? Is the connection CURL context not retryable? Won't exiting here just have the process restart by monit instead?

Reply:
The current implementation doesn't exit here. The intent was to allow context specific logic to determine whether or not to exit based on a response code. I'll remove it for now since it seems like there is never a case where we would want to stop looping on easy_perform

Reply:
Yeah, that sounds good - I can't think of something currently to make us exit, especially as we're in a daemon loop.